config/locales/subscriber_authentication.yml

```diff
@@ -18,6 +18,14 @@ en:
         message: "Enter your email address"
     request_sign_in_token:
       heading: "Check your inbox"
-      confirmation: "We’ve sent an email to %{address}."
-      prompt: Click the link in the email to manage your subscriptions.
-      delay_warning: It can take up to 15 minutes for the email to arrive.
+      description_html: |
+        <p>We’ve sent an email to <strong>%{address}</strong></p>
+        <br>
+        <p>Click the link in the email to confirm your email address.</p>
+        <br>
+        <p>The link will stop working after 7 days.</p>
+      delay_title: Not received an email?
+      delay_html: |
+        <p>Emails sometimes take a few minutes to arrive.</p>
+        <p>If you do not receive an email soon, check your spam or junk folder.</p>
+        <p>If this does not work, <a href="https://www.gov.uk/contact/govuk">contact GOV.UK</a> for help.</p>
```

Comment:
I think you'll want class="govuk-link" on this link. Without it I recall links are almost the same as GOV.UK ones but with a subtle colour difference.

Reply:
Oh yes, I meant to fix that. Thanks for spotting. I've added the `no-visited-state` class, as the target is really an action page, rather than one for someone to read.

The `govuk-link` class appears to be unnecessary if we have the other one and are already in a body-like context, but I've added it in anyway as the two are always seen together.

Reply:
Super. Good shout on the no-visited-state. As I understand .govuk-link uses a different colour palette and is a slightly different blue. I presume the default one is the result of a legacy utility like govuk_frontend_toolkit.

Reply:
> I presume the default one is the result of a legacy utility like govuk_frontend_toolkit

Yeah, it does look like there's a colour state coming from our "core-layout", but I do also recall that we're trying to be less "global", or more specific, about where we apply styles these days.